src/runtime_debug.js

```diff
@@ -133,11 +133,12 @@ function dbg(text) {
   // Avoid using the console for debugging in multi-threaded node applications
   // See https://github.com/emscripten-core/emscripten/issues/14804
   if (ENVIRONMENT_IS_NODE) {
-    fs.writeSync(2, text + '\n');
+    let output = arguments.length === 1 ? text : Array.from(arguments).join(' ');
```

Comment:
I think we are still requiring `var` in most of our JS library code

Reply:
Do you need this check or can you just do `var output = Array.from(arguments).join(' ');`  (i.e. won't it do the right thing for array of length 1?